Software/spartan/src/sensors/lsm6ds33.h

```diff
@@ -164,8 +164,8 @@ namespace spartan {
         uint8_t lsm6Address;
 
         // Note: busID can be either 0 or 1 (can only connect 2 lsm6ds33 modules)
-        LSM6DS33(int busID, int lsm6ID, lsm6Settings settings);
-        LSM6DS33(int busID, int lsm6ID);
+        LSM6DS33(int busID, int lsm6ID, int polling_frequency, lsm6Settings settings);
+        LSM6DS33(int busID, int lsm6IDm, int polling_frequency);
```

Comment:
Is there a reason why we changed the second parameter to `lsm6ID` to `lsm6IDm`? This probably should be producing a compilation error.